Runtime/Core/GameLiftConfiguration.cs

```diff
@@ -11,4 +11,15 @@ public struct GameLiftConfiguration
         public string ApiGatewayEndpoint;
         public bool IsGameLiftAnywhere;
     }
+
```

Comment:
These struct names are too vague. Afaik `GameLiftConfiguration` is a configuration for the server process, while `GameLiftAnywhereConfiguration` describes the compute resource for a fleet (I don't see anything that's specific to anywhere).

Consider updating the naming to `GameLiftServerProcessConfiguration` and `GameLiftComputeConfiguration`. If my assumption is wrong, we should still use clearer naming for distinguishing these structs.